src/lib/journey/callbacks/password/validated-create-password.stories.js

```diff
@@ -61,3 +70,30 @@ Interaction.play = async ({ canvasElement }) => {
   expect(text.type).toEqual('text');
   expect(cb.payload.input[0].value).toBe('myPassword');
 };
+
+export const ConfirmInteraction = Template.bind({});
+
+ConfirmInteraction.args = { ...ConfirmPassword.args };
+
+ConfirmInteraction.play = async ({ canvasElement }) => {
```

Comment:
I had a test failure in this test when running `npm run test:storybook`:

```
Summary of all failing tests
 FAIL  src/lib/journey/callbacks/password/validated-create-password.stories.js (25.121 s)
  ● Callbacks/ValidatedCreatePassword › ConfirmInteraction › play-test

    page.evaluate: StorybookTestRunnerError: 
    An error occurred in the following story. Access the link for full output:
    http://localhost:6006/?path=/story/callbacks-validatedcreatepassword--confirm-interaction&addonPanel=storybook/interactions/panel

    Message:
     expect(received).toBe(expected) // Object.is equality

    Expected: "false"
    Received: "true"


    --------------------------------------------------

    Browser logs:

    warn: <Validated_create_password> was created with unknown prop 'stepMetadata'

    error: {"matcherResult":{"actual":"true","expected":"false","message":"expect(received).toBe(expected) // Object.is equality\n\nExpected: \"false\"\nReceived: \"true\"","name":"toBe","pass":false}}
```

Reply:
Ah, that was an interesting bug. It only failed when the command is run, but not when I test is run manually in the UI. Thanks for catching this.